pkgs/applications/misc/process-compose/default.nix

```diff
@@ -15,7 +15,7 @@ buildGoModule rec {
     sha256 = "TKLLq6I+Mcvdz51m8nydTWcslBcQlJCJFoJ10SgfVWU=";
```

Comment:
You must update the hash otherwise we will keep using the old version

Reply:
:facepalm: thanks.